src/pages/models/[id]/[[...slug]].tsx

```diff
@@ -214,7 +215,7 @@ export default function ModelDetailsV2({
     null;
   const [selectedVersion, setSelectedVersion] = useState<ModelVersionDetail | null>(latestVersion);
   const tippedAmount = useBuzzTippingStore({ entityType: 'Model', entityId: model?.id ?? -1 });
-  const { entities, isLoadingAccess } = useEntityAccessRequirement({
+  const { entities } = useEntityAccessRequirement({
```

Comment:
Can you make `useEntityAccessRequirement` always return true if the user is a moderator. We don't need to be making api calls for this.